src/astt_gui/templates/index.html

```diff
@@ -86,7 +114,7 @@
         <li class="nav-item dropdown">
 
           <a class="nav-link nav-icon" href="#" data-bs-toggle="dropdown">
-            <i class="bi bi-bell"></i><span>Alarms</span><span class="badge bg-danger badge-number">4</span>
+            <i class="bi bi-bell" title="This is an alert button for all warnings, errors and fatal system errors"></i><span>Alarms</span><span class="badge bg-danger badge-number">4</span>
```

Comment:
This looks good and I tested it locally however, we can extend the statements to give more information about what the button does.